analyzers/src/SonarAnalyzer.CSharp.Styling/SonarAnalyzer.CSharp.Styling.csproj

```diff
@@ -27,4 +31,19 @@
     <Using Include="Microsoft.CodeAnalysis.Diagnostics" />
   </ItemGroup>
 
+  <Target Name="CopyBinaries" AfterTargets="Build">
+    <ItemGroup>
+      <BinariesToCopy Include="$(OutputPath)SonarAnalyzer.dll" />
+      <BinariesToCopy Include="$(OutputPath)SonarAnalyzer.CSharp.dll" />
+      <BinariesToCopy Include="$(OutputPath)SonarAnalyzer.CFG.dll" />
+    </ItemGroup>
+    <Copy SourceFiles="@(BinariesToCopy)" DestinationFolder="$(BinariesFolder)" />
+    <!-- Avoid this analyzer to picked up by Java ITs by putting it outside of the binaries folder -->
+    <Copy SourceFiles="$(OutputPath)$(TargetFileName)" DestinationFolder="$(BinariesFolder)\..\internal" />
```

Comment:
This is hacky. Define a new variable in the same place where the BinariesFolder is declared. To keep it all at one place.